File: KTCK_QuanLySinhVien/Them.vb
Public Class Them

    Private _dbAccess As New DataBaseAccess
    Private Sub them_btn_Click(sender As Object, e As EventArgs) Handles them_btn.Click
        If name_TextBox.Text = "" Or class_cbb.Text = "" Or GenderComboBox.Text = "" Or
                phone_TextBox.Text = "" Or email_TextBox.Text = "" Then
            MsgBox("Vui lòng nhập đầy đủ thông tin!!!")
        Else
            Try

                Dim sql As String = "INSERT INTO [SinhVien] VALUES ('" & msv_txt.Text & "',N'" & name_TextBox.Text & "','" & class_cbb.Text & "',N'" & GenderComboBox.Text & "','" & phone_TextBox.Text & "','" & email_TextBox.Text & "')"
                Dim result As Boolean = _dbAccess.ExecuteNoneQuery(sql)

            Catch ex As Exception
                MsgBox(ex.ToString)
            End Try
            MsgBox("Thêm thành công !")
            Me.Close()

        End If
    End Sub

    Private Sub Them_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        loadDataOnCombobox()
    End Sub
    Private Sub loadDataOnCombobox()
        Dim sql As String = "Select * from dbo.Lop"
        Dim dTable As DataTable = _dbAccess.GetDataTable(sql)
        Me.class_cbb.DataSource = dTable
        Me.class_cbb.ValueMember = "TenLop"
        Me.class_cbb.DisplayMember = "TenLop"
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Me.Close()
    End Sub
End Class

File: KTCK_QuanLySinhVien/Them.cs
using System;
using Microsoft.VisualBasic;

namespace KTCK_QuanLySinhVien
{
    public partial class Them
    {

        private DataBaseAccess _dbAccess = new DataBaseAccess();

        public Them()
        {
            InitializeComponent();
        }
        private void them_btn_Click(object sender, EventArgs e)
        {
            if (string.IsNullOrEmpty(name_TextBox.Text) | string.IsNullOrEmpty(class_cbb.Text) | string.IsNullOrEmpty(GenderComboBox.Text) | string.IsNullOrEmpty(phone_TextBox.Text) | string.IsNullOrEmpty(email_TextBox.Text))
            {
                Interaction.MsgBox("Vui lòng nhập đầy đủ thông tin!!!");
            }
            else
            {
                try
                {

                    string sql = "INSERT INTO [SinhVien] VALUES ('" + msv_txt.Text + "',N'" + name_TextBox.Text + "','" + class_cbb.Text + "',N'" + GenderComboBox.Text + "','" + phone_TextBox.Text + "','" + email_TextBox.Text + "')";
                    bool result = _dbAccess.ExecuteNoneQuery(sql);
                }

                catch (Exception ex)
                {
                    Interaction.MsgBox(ex.ToString());
                }
                Interaction.MsgBox("Thêm thành công !");
                Close();

            }
        }

        private void Them_Load(object sender, EventArgs e)
        {
            loadDataOnCombobox();
        }
        private void loadDataOnCombobox()
        {
            string sql = "Select * from dbo.Lop";
            var dTable = _dbAccess.GetDataTable(sql);
            class_cbb.DataSource = dTable;
            class_cbb.ValueMember = "TenLop";
            class_cbb.DisplayMember = "TenLop";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}